cmd/cut-release/main.go

```diff
@@ -72,10 +72,12 @@ func generateTypescriptClientDocs() {
 	}
 }
 
+// TODO(jalextowle): Docs should also be built for `@0x/mesh-browser-lite` or the
```

Comment:
@fabioberger I can add this in this PR. What docs solution do you prefer here?

Reply:
@jalextowle let's remove this version update stages from `cut-release`. We will not be using `lerna publish` which does both the `yarn install` and version updates for us.

Reply:
Just to clarify, does lerna publish also update the version strings in the generated docs?

Reply:
Also, does lerna handle docs generation? It seems like this script should still be used to generate docs.

Reply:
Ah sorry, realized I replied to the wrong comment here. Lerna updates versions in `package.json` but _does not_ generate docs. That part of `cut-release` must be kept.